OrbitGl/GlPanel.h

```diff
@@ -64,4 +67,6 @@ class GlPanel {
   int m_MainWindowWidth;
   int m_MainWindowHeight;
   bool m_NeedsRedraw;
+
+  bool m_IsMouseOver;
```

Comment:
Nit: Newly added members should follow the style is_mouse_over_ (yes, it's inconsistent with what's there, but we are trying to move to Google C++ style over time).